dashboard/db/migrate/20210331171524_add_script_id_to_channel_tokens.rb

```diff
@@ -0,0 +1,7 @@
+class AddScriptIdToChannelTokens < ActiveRecord::Migration[5.2]
+  def change
+    add_column :channel_tokens, :script_id, :integer
+    remove_index :channel_tokens, [:storage_id, :level_id]
+    add_index :channel_tokens, [:storage_id, :level_id, :script_id], unique: true
```

Comment:
I think it would be slightly safer to add the new index before removing the old one. It would be bad if something went wrong and we ended up without either index.